src/libEth/Eth.cpp

```diff
@@ -158,9 +159,9 @@ bool ValidateEthTxn(const Transaction &tx, const Address &fromAddr,
                                gasPriceWei.convert_to<std::string>());
   }
 
-  if (tx.GetGasLimit() < MIN_ETH_GAS) {
+  if (tx.GetGasLimitRaw() < MIN_ETH_GAS) {
```

Comment:
Sorry, I re-read the above, and it looks like it needs to be Raw.  But I'd prefer to use 'Raw' only in the signature calculation code (and even then it will probably use m_coreInfo directly).